liquid_node/jobs/__init__.py

```diff
@@ -126,6 +128,25 @@ def get_job(hcl_path, substitutions={}):
     return output
 
 
+def wait_for_stopped_jobs(stopped_jobs):
```

Comment:
The `gc` command we already have should stop any jobs the user disabled in the config.